hibernate-reactive-quickstart/src/main/java/org/acme/hibernate/reactive/FruitMutinyResource.java

```diff
@@ -25,65 +25,67 @@
 public class FruitMutinyResource {
 
     @Inject
-    Uni<Mutiny.Session> mutinySession;
+    Mutiny.Session mutinySession;
 
     @GET
     public Uni<Fruit[]> get() {
-        return mutinySession.flatMap( session -> session.createNamedQuery( "Fruits.findAll", Fruit.class ).getResultList() )
-                .map( fruits -> fruits.toArray( new Fruit[fruits.size()] ) );
+        return mutinySession
+                .createNamedQuery( "Fruits.findAll", Fruit.class ).getResultList()
+                .onItem().apply( fruits -> fruits.toArray( new Fruit[fruits.size()] ) );
```

Comment:
Could the "get" method return `Uni<List<Fruit>>` rather than needing the `toArray` conversion?

Reply:
I think I tried and didn't work, I'll have a second look.

Reply:
Yep, it works with List. Nice.
I think we should update the ORM test as well: https://github.com/quarkusio/quarkus-quickstarts/blob/development/hibernate-orm-quickstart/src/main/java/org/acme/hibernate/orm/FruitResource.java#L37